### ISSUE TITLE
Testing with test set

### ISSUE THREAD

**Issue by holtvogt** [open-mmlab/mmdetection3d#1784]
My custom dataset is currently organized in a KITTI format:

```
mmdetection3d
├── mmdet3d
├── tools
├── configs
├── data
│   ├── custom
│   │   ├── ImageSets
│   │   ├── testing
│   │   │   ├── velodyne
│   │   ├── training
│   │   │   ├── label
│   │   │   ├── velodyne
```

As mentioned in #1691, the testing config is, e.g. in KITTI dataset, the same as the validation config. When testing with `tools/test.py`, we are basically testing the model on the validation set which already has been used to fine tune the model during the training process, am I right? Just replacing the `split` and `ann_file` property with the corresponding testing name doesn't do the trick here as it can't find the annotations. Do we need to add another label folder inside the `testing` folder or how can we achieve to test on the real test set which the model hasn't seen before?

**Comment by Joaovsky** [comment]
Hey there. According to the Kitti leaderboard submission of the test results, you have to create a split of the training dataset and use that as your new test set (data+labels). Though the problem I have right now is that I can test a pretrained model and get reasonable mAP results but when I use checkpoints I created in my own training (like epoch_40.pth) and use it for testing purposes, mAP is 0 in almost every field (except in one or two where it's like 1.23 or 0.56)...

**Comment by holtvogt** [comment]
I don't understand why one splits the training data set. The point behind the test split is that the model gets unknown data as input

**Comment by holtvogt** [comment]
Yes, I figured that simply defining the testing set as validation set and creating a new dataset using `create_data.py` does indeed do the trick. The corresponding entries in `ImageSets` need to be adjusted as well
